src/metadata/selectors.js

```diff
@@ -309,6 +309,32 @@ export const getCoCByCategoryOptions = createCachedSelector(
     }
 )((_, categoryComboId) => categoryComboId)
 
+/**
+ * @param {*} metadata
+ * @param {*} categoryComboId
+ */
+export const getComputedCategoryOptionsByCatComboId = createCachedSelector(
```

Comment:
Naming is hard. But this computes all the combinations of `categoryOption`s in a `catCombo`, so the result is a list with lists of `categoryOptions`-ids.